addons/civilian/functions/fnc_assignCityCivilian.sqf

```diff
@@ -0,0 +1,25 @@
+#include "script_component.hpp"
+/*
+ * Author: 3Mydlo3
+ * Function assigns city to civilian and vice versa.
+ *
+ * Arguments:
+ * 0: Civilian <OBJECT>
+ * 1: City namespace <CBA_NAMESPACE>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [bob, [bob] call afsk_civilian_fnc_getNearestCity] call afsk_civilian_fnc_assignCityCivilian;
+ *
+ * Public: No
+ */
+
+params ["_civilian", "_cityNamespace"];
+
+// Get city civilians array and pushback new civilian. No need for setVariable as getVariable returns array pointer.
+_cityCivilians = _cityNamespace getVariable QGVAR(CiviliansList);
+_cityCivilians pushBack _civilian;
```

Comment:
`GVAR(CiviliansList)` is public but pushBack will not update it on other clients. Not sure if it's needed everwhere tho.

Reply:
I think it's not needed everywhere, it's only important for server, when civilian movement logic is implemented. I will remove it as global, so in the future it will be easier to troubleshoot any problems.